_docs/render.py

```diff
@@ -103,7 +102,7 @@ def _build_example(contrib: Executable) -> str:
 
     if not callable(func):
         return ""
-    if isinstance(func, MagicFactory):
+    if isinstance(func, partial):
```

Comment:
Is this true for any partial or `hasattr` check should be added?

Reply:
true for any partial